public/style.css

```diff
@@ -36,4 +36,8 @@ ul {
 
 .show-correct-answer {
   border: 3px solid darkgreen;
+}
+
+.show-result{
+  display: none;
 }
```

Comment:
Why are we using this?

Reply:
I create a result button that is hidden by default.